double-check-tmp.md

```diff
@@ -0,0 +1,28 @@
+- No copyright/license mentioned
+  + http://www.jjj.de/fxt/fxtbook.pdf
+  + http://igm.univ-mlv.fr/~mac/REC/text-algorithms.pdf
```

Comment:
Posted on the author's website: http://igm.univ-mlv.fr/~mac/index_en.html No problem linking to it.